<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人体追踪软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="content">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openImage"/>
    <addaction name="openVedio"/>
    <addaction name="openCamera"/>
    <addaction name="closeCamera"/>
    <addaction name="separator"/>
    <addaction name="saveFile"/>
    <addaction name="separator"/>
    <addaction name="closeApp"/>
   </widget>
   <widget class="QMenu" name="about">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="aboutApp"/>
   </widget>
   <addaction name="file"/>
   <addaction name="about"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openImage">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="openVedio">
   <property name="text">
    <string>打开视频</string>
   </property>
  </action>
  <action name="openCamera">
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </action>
  <action name="closeApp">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="aboutApp">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="closeCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
